Tests/IntegrationTests/Components Tests/PayTo/PayToComponentTests.swift

```diff
@@ -60,6 +60,22 @@ class PayToComponentTests: XCTestCase {
         // Then
         XCTAssertNotNil(flowSelectionItem, "Flow selection item should exist")
     }
+    
+    func test_phoneNumberItem_exists() throws {
+        // Given
+        let sut = try PayToComponent(
+            paymentMethod: AdyenCoder.decode(payto),
+            context: Dummy.context
+        )
+
+        setupRootViewController(sut.viewController)
```

Comment:
Please replace it with `viewController.loadIfNeeded()` so this can be an integration not UI test.